atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -1736,7 +1737,15 @@ private boolean rollbackFailedTransactions(
             }
         }
 
-        deleteExecutor.submit(() -> deleteCells(tableRef, keysToDelete));
+        try {
+            deleteExecutor.submit(() -> deleteCells(tableRef, keysToDelete));
```

Comment:
It may help to batch these delete calls. We can union "keys to delete maps" into a single batch before sending these to C* - would be helpful if we submit these calls with a very high rate. We can address that in a separate pr maybe?

Reply:
you wouldn't necessarily want to put them all in the same batch would you? a la our lock request too big etc., but yeah batching seems reasonable

Reply:
Yes, autobatcher would make sense (though I guess not for this PR, as we want to get this out quickly)